opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/services/EncryptionServiceImpl.java

```diff
@@ -0,0 +1,63 @@
+package de.adorsys.opba.protocol.facade.services;
+
+import com.google.crypto.tink.Aead;
+import com.google.crypto.tink.subtle.AesGcmJce;
+import de.adorsys.opba.protocol.api.services.EncryptionService;
+import lombok.SneakyThrows;
+import org.springframework.stereotype.Service;
+
+import javax.crypto.SecretKey;
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.PBEKeySpec;
+import java.security.Provider;
+import java.util.Base64;
+
+@Service
+public class EncryptionServiceImpl implements EncryptionService {
```

Comment:
Can be simplified to `@RequiredArgsConstructor`

Reply:
ok